src/CMakeLists.txt

```
@@ -1,10 +1,13 @@
-cmake_minimum_required(VERSION 3.3)
+cmake_minimum_required(VERSION 3.2)
 project(QZXing)
 
-find_package(Qt5 COMPONENTS Core REQUIRED)
-find_package(Qt5 COMPONENTS Gui REQUIRED)
-find_package(Qt5 COMPONENTS Multimedia )
-find_package(Qt5 REQUIRED Svg Quick QuickControls2)
+set(QT_COMPONENTS
+    Core Gui Multimedia Svg Quick QuickControls2
```

```suggestion
    Core Gui Svg Quick QuickControls2
```
The multimedia should be added only when QZXING_MULTIMEDIA is enabled.

---

fixed in new commit